css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-between;
+  max-height: 100vh;
+}
+
+.row {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-evenly;
+  flex-basis: auto;
+}
```

Comment:
`flex-basis` should only go on the children of the flex container, not the flex container itself. You should remove this here.